src/indexer/index_writer.rs

```diff
@@ -974,7 +974,7 @@ mod tests {
         assert_eq!(
             format!("{:?}", index_writer.get_merge_policy()),
             "LogMergePolicy { min_num_segments: 8, max_docs_before_merge: 10000000, min_layer_size: 10000, \
-             level_log_size: 0.75 }"
+             level_log_size: 0.75, max_del_docs_pct: 100 }"
         );
```

Comment:
I would prefer
max_del_docs_percentage

Reply:
Alternatively `max_del_docs_ratio: f32`?

Reply:
I think max_del_docs_percentage is a little bit easier to understand. To be more precise maybe
`del_docs_percentage_before_merge`
 
I think we can drop the max part?

Reply:
`del_docs_percentage_before_merge` is more consistent with the current naming, renamed!